lib/vsc/mympirun/mpi/mpich.py

```diff
@@ -42,16 +45,17 @@ class MVAPICH2Hydra(MPI):
 
     PASS_VARIABLES_CLASS_PREFIX = ['MV2', 'HYDRA']
 
-    MPIEXEC_TEMPLATE_PASS_VARIABLE_OPTION = "-envlist %(commaseparated)s"
+    MPIEXEC_TEMPLATE_PASS_VARIABLE_OPTION = "-x %(name)s=%(value)s"
```

Comment:
what if `value` has whitespace? (and why dod you change this; is envlist not working anymore?)